cmd/root.go

```diff
@@ -214,6 +214,9 @@ default values.`,
 func initConfig() {
 	Context = context.Background()
 	var err error
+	if len(configurationFile) == 0 {
+		configurationFile = os.Getenv("ROSETTA_CONFIGURATION_FILE")
```

Comment:
@sosedoff, can you make this a constant? Would also appreciate a little comment blurb saying what we are doing (as the double `if len(configurationFile) == 0 {` could be confusing to others...maybe indicating that `os.Getenv` defaults to `""`)!

Reply:
Yep, will do. I just wanted to see if you would be interested in the change in the first place. I will update the readme as well.

Reply:
👍  Love the idea!